tests/openapi/openapi_integration/test_snapshot.py

```diff
@@ -361,3 +361,6 @@ def test_snapshot_invalid_file_uri():
     )
     assert response.status_code == 400
     assert response.json()["status"]["error"] == "Bad request: Snapshot file \"/whatever.snapshot\" does not exist"
+
+    # Give file system some time to clean up temporary snapshot files
+    sleep(1)
```

Comment:
I am having one last issue with this fix :sweat_smile: 

The `test_snapshot_invalid_file_uri` test does not create any valid snapshot as far as I can tell.
Therefore it is slightly weird having to wait for cleaning the temporary snapshot files at the end.

Makes sense?

Reply:
You're right, it is very confusing.

After another look, it's not `test_snapshot_invalid_file_uri` that creates files and fails, but other snapshot tests before it. But before starting `test_snapshot_invalid_file_uri` we list/delete these same snapshot files.

I'll update the fix putting stuff in the right place so that it makes more sense.

Thanks for the careful consideration!